src/main/java/hudson/plugins/brakeman/scanners/AbstractBrakemanScanner.java

```diff
@@ -0,0 +1,16 @@
+package hudson.plugins.brakeman.scanners;
+
+import hudson.plugins.analysis.core.ParserResult;
+
+/**
+ * A Java class that representats a JSON Scanner of the Brakeman Output file
```

Comment:
I legit copied this documentation and changed (most) of the names to fit this. An abstract class would probably be better